fpga_interchange/interchange_capnp.py

```diff
@@ -729,8 +729,8 @@ def to_physical_netlist(phys_netlist_capnp):
 
     for physical_cell in phys_netlist_capnp.physCells:
         phys_netlist.add_physical_cell(
-            strs[physical_cell.cellName],
-            PhysicalCellType(physical_cell.physType))
+            strs[physical_cell.cellName], PhysicalCellType[first_upper(
+                str(physical_cell.physType))])
```

Comment:
Is the `str` here required?  I don't believe so?

Reply:
Without that I get a:
```
TypeError: 'capnp.lib.capnp._DynamicEnum' object is not subscriptable
```